api/src/main/java/org/openmrs/module/attachments/AttachmentsContext.java

```diff
@@ -196,6 +210,43 @@ protected Boolean getBooleanByGlobalProperty(String globalPropertyName) {
 		return BooleanUtils.toBoolean(globalProperty);
 	}
 	
+	protected Concept getConceptByGlobalProperty(String globalPropertyName) {
+		String globalProperty = administrationService.getGlobalProperty(globalPropertyName);
+		Concept concept = conceptService.getConceptByUuid(globalProperty);
+		if (concept == null) {
+			throw new IllegalStateException("Configuration required: " + globalPropertyName);
```

Comment:
Probably better to use our `APIException` class for these, e.g.,

```suggestion
			throw new APIException("Configuration required: " + globalPropertyName);
```

These are configuration issues, not programming issues.